.github/workflows/publish-cdk-command-manually.yml

```diff
@@ -19,9 +19,6 @@ on:
           - minor
           - patch
         required: true
-      skip-publish-test:
```

Comment:
Outside of the scope of this PR but wasn't used anymore since we removed earlier